<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="login">
       <widget class="QLabel" name="id_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nimbus Sans L</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>National ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_id">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="newuser_button">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>New user</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login_button">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>160</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="register_2">
       <widget class="QLabel" name="name_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="id_label_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>70</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>National ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="age_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_name">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>281</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_id_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>280</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_age">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>281</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="register_button">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>280</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Register</string>
        </property>
       </widget>
       <widget class="QLabel" name="gender_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>190</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
       <widget class="QComboBox" name="gender_combox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>220</y>
          <width>281</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Noto Sans CJK KR DemiLight</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="back_button_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>280</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="main">
       <widget class="QLabel" name="welcome_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>381</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deposit_button">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>80</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>DEPOSIT</string>
        </property>
       </widget>
       <widget class="QPushButton" name="withdraw_button">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>160</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>WITHDRAW</string>
        </property>
       </widget>
       <widget class="QPushButton" name="checkbalance_button">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>80</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>CHECK BALANCE</string>
        </property>
       </widget>
       <widget class="QPushButton" name="undo_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>240</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string> UNDO</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/undo.png</normaloff>
          <normalon>:/images/undo.png</normalon>:/images/undo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="redo_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>240</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string> REDO</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/redo.png</normaloff>
          <normalon>:/images/redo.png</normalon>:/images/redo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="statistics_button">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>160</y>
          <width>150</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>STATISTICS</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="statistics">
       <widget class="QCustomPlot" name="custom_plot" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>451</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="back_button">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>270</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="amount">
       <widget class="QLineEdit" name="input_amount">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="amount_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>151</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Nimbus Sans L</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ok_button">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>170</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_button_3">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="response">
       <widget class="QLabel" name="response_label">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>261</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="ok_button_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 94, 135);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>20</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>exit_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
